subjects/reverse.en.md

```diff
@@ -42,17 +42,8 @@ import (
     "fmt"
 )
 
-func pushBack(n *NodeAddL, num int) *NodeAddL{
-
-}
-
 func main() {
-	num1 := &piscine.NodeAddL{Num: 1}
-	num1 = pushBack(num1, 3)
-	num1 = pushBack(num1, 2)
-	num1 = pushBack(num1, 4)
-	num1 = pushBack(num1, 5)
-
+	// num1 = 1 -> 2 -> 3 -> 4 -> 5
 	result := piscine.Reverse(num1)
```

Comment:
The exercise `reverse` should not have piscine.NodeAddL or piscine.Reverse because the functions and the structure are made in the same package, `package main`.